### ISSUE TITLE
Please update your Dockerfile to not use a hardcoded 1000 uid

### ISSUE THREAD

**Issue by electrofloat** [psi-4ward/psitransfer#258]
Using id 1000 is very restrictive.

Would be a tons better if one could just give the uid/gid through env variables like what linuxserver.io does:
https://docs.linuxserver.io/general/understanding-puid-and-pgid

**Comment by psi-4ward** [comment]
That's the standard of node:alpine. But feel free to make some suggestions

**Comment by electrofloat** [comment]
Yes. That's why I'm suggesting to do the same as linuxserver does. 
Use PUID and PGID env variables (which will be given by the user creating the container with --env), and either create a new user in your image with those ids for ex.:
```
groupmod -o -g "$PGID" abc
usermod -o -u "$PUID" abc
.
.
chown abc:abc /folder_used_by_your_app
```
or change the uid/gid of use node to the one provided in the env variables, for ex.:
```
usermod -u "$PUID" node
groupmod -g "$PGID" nodegroup
.
.
find / -group "$PGID" -exec chgrp -h nodegroup {} \;
find / -user "$PUID" -exec chown -h node {} \;
```
The last one of course assumes that the numbers given through env are available in the image itself, so probably the first one is safer to use.

**Comment by psi-4ward** [comment]
If you like prep a pr
